src/arrayMethodSort.js

```diff
@@ -3,9 +3,29 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    return String(a) > String(b);
+  }) {
+    let count;
```

Comment:
```suggestion
  [].__proto__.sort2 = function(callback) {
    let count;
    const compareFunction = callback || (a, b) => String(a) > String(b);
```
This is much easier to debug.